### ISSUE TITLE
Questions about loss

### ISSUE THREAD

**Issue by omg777** [marvis/pytorch-yolo2#93]
In training message, 
`64: nGT 151, recall 63, proposals 52, loss: x 105.755363, y 96.717644, w 1591.836548, h 1606.528076, conf 7463.996582, cls 446.882355, total 11311.716797`

What`s the meaning of nGT, proposals, x,y,w,h, conf, cls and total ?

**Comment by DreadPiratePsyopus** [comment]
Just read the code/ paper.

nGT number of Ground Truths.
x, y, w,h coordinates, width and height according to paper.
conf confidence of a class
cls the class
total the sum of all the losses.
